src/AboutScreen/about.jsx

```diff
@@ -5,84 +5,92 @@ import { useSelector } from "react-redux";
 import { getVersion, getBuildNumber } from "react-native-device-info";
 import colors from "../common/colors";
 import STRINGS from "../common/localization";
-import styles from "./styles";
+import { styles, nightStyles } from "./styles";
 import constant from "../common/constant";
 import useHeader from "./hooks/useHeader";
 
 function AboutScreen({ navigation }) {
   const { isNightMode, isStatusBar } = useSelector((state) => state);
   useHeader(navigation);
+  const { backgroundColor } = nightStyles(isNightMode);
+  const { TOOLBAR_COLOR_ALT2 } = colors;
+  const { nightMode, SGTitle, margin10, margin, singleLine, leftContainer, underlayColor } = styles;
+  const {
+    CREATED_BY,
+    SUNDAR_GUTKA,
+    ABOUT_1,
+    ABOUT_2,
+    ABOUT_3,
+    ABOUT_4,
```

Comment:
Can we use more descriptive names here, 
e.g. Instead of ABOUT_1, it can be ABOUT_GREETING
ABOUT_2 can be ABOUT_MORE_INFO and so on..

Reply:
fixed